### DISCUSSION TITLE
Market > Trades > History: add "max N hours/days in the past" optional field

### DISCUSSION THREAD

**Discussion by GordianLN** [bisq-network/bisq#5830] [Ideas]
When I want to check what have been the highest and lowest price offers lately, to decide how far I should go with my deviation %, I often find myself exporting to CSV, importing in spreadshit, delete all entries older than X, and then sort by price.
Needless to say, it is quite a long procedure.
Also, usually you don't need to show eveyrhing that went down in history up to the launch of Bisq network, just the last few days to check on the recent market.

Ordering by price, as is, will be impacted by trades that went down when bitcoin, lucky few, was still at  a piece.

I think that adding a field where you can input a max number of days, hours, anything (maybe enabling smart formatting like `24h`, `7d`, `12M`, `3600s`, `60m`, `4w`, you know what I mean) could at the same time make the trades history even more useful, and cut down on resources, as that numeric field would be saved between restarts, unless you clear it out and it goes back to displaying the whole history.

**Comment by pazza83** [comment]
Mentioned this here: https://github.com/bisq-network/bisq/discussions/5799#discussioncomment-1646150
